spring-cloud-deployer-resource-support/src/main/java/org/springframework/cloud/deployer/resource/support/DelegatingResourceLoader.java

```diff
@@ -137,12 +137,10 @@ public Resource getResource(String location) {
 				return new FileSystemResource(cachedResource);
 			}
 		}
-		catch (URISyntaxException e) {
```

Comment:
If the Exception is going to be applicable more generally than "not-found" problems, then it should probably be renamed to something that is similarly general like `ResourceLoadingException`. Although at that point, I think it's less convincing that adding a new Exception is helpful.